webserver/views/datasets.py

```diff
@@ -286,10 +299,11 @@ def create_service():
 def import_csv():
     form = forms.DatasetCSVImportForm()
     if form.validate_on_submit():
+        [description, classes] = _parse_dataset_csv(request.files[form.file.name])
         dataset_dict = {
             "name": form.name.data,
-            "description": form.description.data,
-            "classes": _parse_dataset_csv(request.files[form.file.name]),
+            "description": description if not None else form.description.data,
```

Comment:
did you mean to say `description if description not None`? You can also say `description if description else form...`